src/appkit.cs

```diff
@@ -2125,6 +2125,10 @@ public interface NSButton : NSAccessibilityButton {
 		[Mac (10,12)]
 		[Export ("imageHugsTitle")]
 		bool ImageHugsTitle { get; set; }
+
+		[Mac (10,5)]
```

Comment:
trivial but we generally don't add availability attributes for versions lower than our minimum support (e.g. 10.7)